utils/registry/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "utils"
```

Comment:
Let's call it `registry.wasm`

Reply:
I have changed it. Because '.' is invalid character, I replaced it with '_'.